docs/libcurl/opts/CURLOPT_SSL_VERIFYPEER.3

```diff
@@ -57,6 +57,15 @@ man-in-the-middle the communication without you knowing it. Disabling
 verification makes the communication insecure. Just having encryption on a
 transfer is not enough as you cannot be sure that you are communicating with
 the correct end-point.
+
+NOTE: even when this option is disabled, curl will still try to verify
+the certificate, though errors are ignored. curl default settings in
+some distributions might use quite a large file as a default setting
+for \fICURLOPT_CAINFO(3)\fP, so certificate verification can become quite
+expensive, especially when dealing with many connections. Thus, in some
+situations, you might want to disable verification fully to save resources
+by setting \fICURLOPT_CAINFO(3)\f to NULL - but please also consider the
+warning above!
```

Comment:
I think it should rather say that it still might (depending on TLS backend) *load* the certificates. I don't think saying "it will verify but ignore the result" helps the user even if it technically is true. This is an area I want to make sure people will not misunderstand so I rather not mention "verify" where it doesn't actually verify...